content/tokio/tutorial/async.md

```diff
@@ -542,6 +544,21 @@ impl MiniTokio {
             task.poll();
         }
     }
+
+    // Create a new crossbeam::channel and sets it to the new
+    // MiniTokio instance.
```

Comment:
Doc-comments should match those in [tutorial-code](https://github.com/tokio-rs/website/blob/48bff2dcceb4334b2cb1b8aeedf4dafe8c2134c8/tutorial-code/mini-tokio/src/main.rs#L68-L85). They should also use `///` rather than `//`.